ingredients/DECam/DRP.yaml

```diff
@@ -173,21 +173,53 @@ subsets:
       expression if any later steps will also be run, because downstream steps
       require full visits and 'tract' and 'patch' constraints will always
       select partial visits that overlap that region.
-  step2:
+  step2a:
     subset:
       - consolidatePreSourceTable
       - consolidateVisitSummary
+    description: >
+      visit-level tasks. Permissible dataquery constraints: visit
```

Comment:
'Visit' here should be capitalized.

For those who don't like 'permissible', would 'allowed' be more friendly to use here?

Would it be useful to list 'none' in addition to 'visit', if it's also okay to run this step globally?

Reply:
Ken suggested `permitted`?  We use  `allowed` roughly 2x more frequently than `permitted` in the stack